composer.json

```diff
@@ -179,6 +179,7 @@
     "conflict": {
         "doctrine/doctrine-bundle": "2.3.0",
         "jms/serializer-bundle": "3.9.0",
+        "sonata-project/block-bundle": "4.13.0",
```

Comment:
I believe that after https://github.com/Sylius/Sylius/pull/14020 there is no need for this conflict